snuba/datasets/schemas/tables.py

```diff
@@ -203,6 +221,9 @@ def __get_local_source_table_name(self) -> str:
     def __get_local_destination_table_name(self) -> str:
         return self._make_test_table(self.__local_destination_table_name)
 
+    def _get_local_engine(self) -> str:
+        pass
```

Comment:
I thought I wrote a comment about this earlier but I can't find it (so either I didn't, or I totally missed the response, or I put it on a different diff and forgot): why is `_get_local_engine` defined here? I don't see it used (and it's return value won't type check.) I also don't think it is part of the class hierarchy ay longer, from what I can tell?

Reply:
This is a mistake. 
I moved _get_local_engine into MergeTreeSchema. It is not in TableSchema anymore. I just forgot to remove the method from here.